qemu/tests/seabios.py

```diff
@@ -34,24 +35,73 @@ def boot_menu_check():
         return (len(re.findall(boot_menu_hint,
                                get_output(seabios_session))) > 1)
 
-    error.context("Start guest with sga bios")
     vm = env.get_vm(params["main_vm"])
     # Since the seabios is displayed in the beginning of guest boot,
     # booting guest here so that we can check all of sgabios/seabios
     # info, especially the correct time of sending boot menu key.
     vm.create()
+    seabios_session = vm.logsessions['seabios']
+    login_timeout = float(params.get("login_timeout", 360))
+    bios_bin_dic = {'rhel6': 'bios.bin', 'rhel7': 'bios-256k.bin'}
+    env_process.preprocess_vm(test, params, env, params.get("main_vm"))
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login(timeout=login_timeout)
+    output = vm.monitor.info("roms")
+    match_bin = re.search(bios_bin_dic['rhel6'], output) or re.search(bios_bin_dic['rhel7'], output)
```

Comment:
This command is same as re.search('bios.bin'|'bios-256k.bin'), if rhel7 return bios.bin, it passed too.

Reply:
@sitoliu I have updated with match_bin = re.search('bios.bin|bios-256k.bin', output)